garrysmod/gamemodes/terrortown/gamemode/traitor_state.lua

```diff
@@ -179,6 +179,7 @@ local function force_spectate(ply, cmd, arg)
       end
    end
 end
-concommand.Add("ttt_spectate", force_spectate)

```

Comment:
Whoops, meant to put it back in because I thought of that while doing some of the other stuff.